pkgs/development/compilers/dotnet/update.sh

```diff
@@ -92,21 +92,28 @@ aspnetcore_packages () {
     # This should not happend in minor or bugfix updates, but probably happens
     # with every new major .NET release.
     local pkgs=( \
-      "Microsoft.AspNetCore.App.Ref" \
       "Microsoft.AspNetCore.App.Runtime.linux-arm" \
       "Microsoft.AspNetCore.App.Runtime.linux-arm64" \
-      "Microsoft.AspNetCore.App.Runtime.linux-musl-arm" \
       "Microsoft.AspNetCore.App.Runtime.linux-musl-arm64" \
       "Microsoft.AspNetCore.App.Runtime.linux-musl-x64" \
       "Microsoft.AspNetCore.App.Runtime.linux-x64" \
-      "Microsoft.AspNetCore.App.Runtime.osx-arm64" \
       "Microsoft.AspNetCore.App.Runtime.osx-x64" \
       "Microsoft.AspNetCore.App.Runtime.win-arm" \
       "Microsoft.AspNetCore.App.Runtime.win-arm64" \
       "Microsoft.AspNetCore.App.Runtime.win-x64" \
       "Microsoft.AspNetCore.App.Runtime.win-x86" \
     )
 
+    # Packages that only apply to .NET 5 and up
+    if [[ ! ( "$version" =~ ^[1-3]\. ) ]]; then
```

Comment:
Can we make this comparison less brittle?

Reply:
Hmm, I don't think this comparision is any more brittle than other ones in this script. I also don't really have an idea how to improve it, so suggestions welcome :)

Reply:
Maybe we can `nix eval` something like `lib.versionOlder`?

Reply:
That sounds like a better idea and less likely to break.

Reply:
For reference, that should also happen on line 240. Should probably create a `versionOlder` function in bash that does the `nix eval` to reduce code repetition.

Reply:
Ok, I updated the checks to use `versionOlder`.